ext/libgav1_android.sh

```diff
@@ -0,0 +1,36 @@
+# This script will build libgav1 for the default ABI targets supported by
+# android. You must pass the path to the android NDK as a parameter to this
+# script.
+#
+# Android NDK: https://developer.android.com/ndk/downloads
+#
+# The git tag below is known to work, and will occasionally be updated. Feel
+# free to use a more recent commit.
+
+if [ $# -ne 1 ]; then
+  echo "Usage: ${0} <path_to_android_ndk>"
+  exit
+fi
+git clone -b v0.17.0 --depth 1 https://chromium.googlesource.com/codecs/libgav1
```

Comment:
you might want to add the same comment here about updating libgav1.cmd

Reply:
Done